lib/components/parallax_component.dart

```diff
@@ -51,7 +51,11 @@ class ParallaxLayer {
 
   bool loaded() => _image != null;
 
-  Vector2 currentOffset() => _scroll;
+  /// FIXME: added this as there are a lot of things being set in resize first time that
+  /// are being used in update
+  bool sized = false;
```

Comment:
Added this. Otherwise, `update` would be bloated with null checks for `_scale`, `_imagesSize`, `_paintArea`